core/error-handler.ts

```diff
@@ -1,6 +1,7 @@
-// import { toast } from 'react-toastify';
+// import toast from "react-hot-toast";
 
-export default (error: string) => {
+export default (error: string): null => {
```

Comment:
I prefer to export a class, later maybe we have `handleError` `handleWarning` and other type of operations

Toast.handleError() is more descriptive of what the function is doing than just `handleError()`